packages/sdk-personal/src/index.ts

```diff
@@ -11,10 +12,6 @@ const createPersonalEvent = (
   isPHI: boolean
 ): SendEvent => {
   const basicEvent = internalUtils.createBasicEvent(event)
-  if (!basicEvent) {
```

Comment:
Why we have removed this ?

Reply:
because we have check in `capturePersonal` that checks for event name. So we can't trigger this if statement anymore